skyway_app/static/map.js

```diff
@@ -9,6 +10,15 @@ function askForUserLocation(){
         lng: position.coords.longitude
       };
       map.setCenter(pos)
+      var image = '/static/homeicon.png';
+      console.log("home icon")
+      image.height = image.height/4
```

Comment:
Seems better to set a specific size. If you swap the image to a different image source, you want the size to be the same, not proportional to the original image